src/BuildIntegration/Microsoft.NETCore.Native.Unix.props

```diff
@@ -70,7 +70,8 @@ See the LICENSE file in the project root for more information.
       <LinkerArg Include="-luuid" Condition="'$(TargetOS)' != 'OSX'" />
       <LinkerArg Include="-lrt" Condition="'$(TargetOS)' != 'OSX'" />
       <LinkerArg Include="-licucore" Condition="'$(TargetOS)' == 'OSX'" />
-      <LinkerArg Include="-shared" Condition="'$(NativeLib)' == 'Shared'" />
+      <LinkerArg Include="-dynamiclib" Condition="'$(TargetOS)' == 'OSX' and '$(NativeLib)' == 'Shared'" />
```

Comment:
I though that `-shared` works on OSX too. Just for my education - why did you change this to `-dynamiclib`?

Reply:
Oh yeah I know it works. I guess I was just trying to be as platform accurate as possible, most Apple docs use `-dynamiclib`

Reply:
Ok, that's fine.